### ISSUE TITLE
Email submitter with approved submission

### ISSUE THREAD

**Issue by nathanshammah** [unitaryfund/metriq-app#683]
Right now the submitter of a new submission receives an email telling them that the submission is under review. 
They do not receive any further communication via email about it. It would be nice to also send an email once the submission is approved and published.

It could include a link to the Tweet that is triggered, will open another issue about it.

**Comment by WrathfulSpatula** [comment]
This was addressed with a administrative script in the back end. We can close the issue, and I'll show the team at the weekly meeting.

**Comment by nathanshammah** [comment]
Fantastic!
